### PR TITLE
Implementation Module 2, Show information of a Movie

### PR DESCRIPTION
**Pull request by SavoGK**
- Implementation of class Movie, this have the information of a movie (title, original Title, thumbnail, overview, vote averaged genres and counter likes).
- Implementation class Movie show, this take the information of a movie and show it. 
- Implementation Like Button, this button take the information about how many like have the movie and show it. If you press it, the number of like increases. 

Screenshot_20230928_122225

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
...
-    );
+            appBar: AppBar(
+              backgroundColor: Colors.black87,
+              title: Text(marioBros.originalTitle),
+            ),
+            body: SafeArea(
+                top: true,
+                bottom: true,
+                left: true,
+                right: true,
```
Comment (Kevin-Guillen) [inline lib/main.dart]: this values of SafeArea are true by default , you can remove them

File: lib/main.dart [hunk 2]
```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
...
+                child: Center(
+                  child: ShowMovie(marioBros),
+                ))));
```
Comment (Kevin-Guillen) [inline lib/main.dart]: commas and format code

File: lib/main.dart [hunk 3]
```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
...
-      required this.originalTitle,
-      required this.thumbnail,
-      required this.poster,
-      required this.overview,
-      required this.voteAverage,
-      required this.genres,
-      required this.countLikes,
-    });
-  }
+class ShowMovie extends StatelessWidget {
```
Comment (Kevin-Guillen) [inline lib/main.dart]: Lets find another name to this class , something that can represent an object , not a functionality

File: lib/main.dart [hunk 4]
```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
...
-          SingleChildScrollView( // this permit can be scroll
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        body: Container(
+      color: Colors.black87,
+      child: Column(
+        children: <Widget>[
+          Image.asset(
+            // Top Image
```
Comment (Kevin-Guillen) [inline lib/main.dart]: remove comment

File: lib/main.dart [hunk 5]
```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
...
+            movie.thumbnail,
+          ),
+          Row(
+            // Score and title
```
Comment (Kevin-Guillen) [inline lib/main.dart]: remove comment

File: lib/main.dart [hunk 6]
```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
...
+            children: [
+              Padding(
+                padding: const EdgeInsets.all(8.0),
+                child: Card(
+                  // show the score
```
Comment (Kevin-Guillen) [inline lib/main.dart]: remove comment

File: lib/main.dart [hunk 7]
```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
...
+                  shape: RoundedRectangleBorder(
+                    borderRadius: BorderRadius.circular(80),
+                    // made it circular
```
Comment (Kevin-Guillen) [inline lib/main.dart]: remove comment

File: lib/main.dart [hunk 8]
```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
...
+                    side: const BorderSide(
+                        width: 5, color: Colors.indigo), // add a border
```
Comment (Kevin-Guillen) [inline lib/main.dart]: constant value , for this if the value is used just one time you cant declarate the constant in the same file outside the build , if not , you should take all those constant that you use more that once to a separated file , like lib -> utils-> ui_constants

File: lib/main.dart [hunk 9]
```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
...
+                  ),
+                  color: Colors.blueAccent,
+                  child: Padding(
+                      padding: const EdgeInsets.all(15),
+                      child: Text(
+                        '${movie.voteAverage}',
+                        textScaleFactor: 2,
```
Comment (Kevin-Guillen) [inline lib/main.dart]: constant

File: lib/main.dart [hunk 10]
```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
...
+                  // show the score
+                  shape: RoundedRectangleBorder(
+                    borderRadius: BorderRadius.circular(80),
+                    // made it circular
+                    side: const BorderSide(
+                        width: 5, color: Colors.indigo), // add a border
+                  ),
+                  color: Colors.blueAccent,
+                  child: Padding(
+                      padding: const EdgeInsets.all(15),
```
Comment (Kevin-Guillen) [inline lib/main.dart]: constant

File: lib/main.dart [hunk 11]
```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
...
+                      child: Text(
+                        '${movie.voteAverage}',
+                        textScaleFactor: 2,
+                      )),
+                ),
+              ),
+              Expanded(
+                child: Text(
+                  // Title
```
Comment (Kevin-Guillen) [inline lib/main.dart]: remove comment

File: lib/main.dart [hunk 12]
```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
...
+                  movie.title,
+                  style: const TextStyle(
+                      color: Colors.white,
+                      fontWeight: FontWeight.bold,
+                      fontSize: 20.0),
```
Comment (Kevin-Guillen) [inline lib/main.dart]: constant

File: lib/main.dart [hunk 13]
```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
...
+                ),
+              ),
+            ],
+          ),
+          SingleChildScrollView(
+            // this permit can be scroll
             scrollDirection: Axis.horizontal, // in the horizontal axis
             padding: const EdgeInsets.fromLTRB(10.0, 0.0, 10.0, 0.0),
```
Comment (Kevin-Guillen) [inline lib/main.dart]: you can change this to EdgeInsets.symmetric(horizontal: 10.0)  and add 10.0 to constants

File: lib/main.dart [hunk 14]
```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
...
             child: Row(
               children: [
-                const Text(
-                    'Genres',
+                const Text('Genres',
```
Comment (Kevin-Guillen) [inline lib/main.dart]: constant value , as well as the int/double constant values that i commented , you can add a file called string_constant where you can put all this strings as constants

File: lib/main.dart [hunk 15]
```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
...
                     style: TextStyle(
                       color: Colors.white,
                       fontSize: 16,
```
Comment (Kevin-Guillen) [inline lib/main.dart]: constant value

File: lib/main.dart [hunk 16]
```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
...
-                    )
+                    )),
+                const SizedBox(
+                  width: 10,
```
Comment (Kevin-Guillen) [inline lib/main.dart]: constaant value

File: lib/main.dart [hunk 17]
```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
...
-                              fontSize: 15,
-                          ),
-                        ),
-                        onSelected: (b) {}),
-                    )).toList()),
+                    mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                    children: movie.genres
+                        .map((genre) => Padding(
+                              padding: const EdgeInsets.all(4.0),
+                              child: FilterChip(
```
Comment (Kevin-Guillen) [inline lib/main.dart]: you can replace this with a container with the same decoration

File: lib/main.dart [hunk 18]
```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
...
+                                  // i used this to show the genres, normally is used to checkbox
+                                  shape: RoundedRectangleBorder(
+                                    borderRadius: BorderRadius.circular(40),
```
Comment (Kevin-Guillen) [inline lib/main.dart]: constant

File: lib/main.dart [hunk 19]
```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
...
-                          ),
-                        ),
-                        onSelected: (b) {}),
-                    )).toList()),
+                    mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                    children: movie.genres
+                        .map((genre) => Padding(
+                              padding: const EdgeInsets.all(4.0),
+                              child: FilterChip(
+                                  // i used this to show the genres, normally is used to checkbox
```
Comment (Kevin-Guillen) [inline lib/main.dart]: remove comment

File: lib/main.dart (hunk 18 above)
Comment (Kevin-Guillen) [inline lib/main.dart]: constant

File: lib/main.dart [hunk 20]
```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
...
+                                  shape: RoundedRectangleBorder(
+                                    borderRadius: BorderRadius.circular(40),
+                                    side: const BorderSide(
+                                        width: 2, color: Colors.indigo),
```
Comment (Kevin-Guillen) [inline lib/main.dart]: constant

File: lib/main.dart [hunk 21]
```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
...
+                                  ),
+                                  backgroundColor: Colors.black87,
+                                  label: Text(
+                                    genre,
+                                    style: const TextStyle(
+                                      color: Colors.white54,
+                                      fontSize: 15,
```
Comment (Kevin-Guillen) [inline lib/main.dart]: constant

File: lib/main.dart [hunk 22]
```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
...
+                                    ),
+                                  ),
+                                  onSelected: (b) {}),
```
Comment (Kevin-Guillen) [inline lib/main.dart]: if you replace filterchip with a container you wont need this

File: lib/main.dart [hunk 23]
```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
...
-             ),
-           ),
-           LikeButton(movie.countLikes),
-         ],
-       ),
-       )
-      );
-    }
+          const Padding(
+            padding: EdgeInsets.all(10.0),
```
Comment (Kevin-Guillen) [inline lib/main.dart]: constant

File: lib/main.dart [hunk 24]
```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
...
+            child: Text(
+              'Description',
```
Comment (Kevin-Guillen) [inline lib/main.dart]: constant

File: lib/main.dart [hunk 25]
```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
...
+              style: TextStyle(
+                  color: Colors.white,
+                  fontWeight: FontWeight.w600,
+                  fontSize: 16.0),
```
Comment (Kevin-Guillen) [inline lib/main.dart]: constant

File: lib/main.dart [hunk 26]
```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
...
+            ),
+          ),
+          Padding(
+            padding: const EdgeInsets.all(10.0),
```
Comment (Kevin-Guillen) [inline lib/main.dart]: constant

File: lib/main.dart [hunk 27]
```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
...
+            child: Text(
+              movie.overview,
+              style: const TextStyle(
+                  color: Colors.white54,
+                  fontWeight: FontWeight.w600,
+                  fontSize: 14.5),
```
Comment (Kevin-Guillen) [inline lib/main.dart]: constant

**Review by Kevin-Guillen — CHANGES_REQUESTED** [review]
Great looking UI , as a general comment, lets avoid having all the widgets built in main and  abstract them into different files.

File: macos/Flutter/GeneratedPluginRegistrant.swift [hunk 28]
```diff
@@ -5,6 +5,10 @@
 import FlutterMacOS
 import Foundation
 
+import shared_preferences_foundation
+import sqflite
 
 func RegisterGeneratedPlugins(registry: FlutterPluginRegistry) {
+  SharedPreferencesPlugin.register(with: registry.registrar(forPlugin: "SharedPreferencesPlugin"))
+  SqflitePlugin.register(with: registry.registrar(forPlugin: "SqflitePlugin"))
```
Comment (Kevin-Guillen) [inline macos/Flutter/GeneratedPluginRegistrant.swift]: What is the purpose of this?

File: lib/main.dart [hunk 29]
```diff
@@ -1,125 +1,218 @@
+import 'package:shared_preferences/shared_preferences.dart';
+import 'package:sqflite/sqflite.dart';
+import 'package:provider/provider.dart';
```
Comment (matiasilveiro-gl) [inline lib/main.dart]: Why these imports? You don't need them for this PR

File: lib/main.dart [hunk 30]
```diff
@@ -1,125 +1,218 @@
...
-  // This widget is the home page of your application. It is stateful, meaning
-  // that it has a State object (defined below) that contains fields that affect
-  // how it looks.
-
-  // This class is the configuration for the state. It holds the values (in this
-  // case the title) provided by the parent (in this case the App widget) and
-  // used by the build method of the State. Fields in a Widget subclass are
-  // always marked "final".
-
+class Movie {
```
Comment (matiasilveiro-gl) [inline lib/main.dart]: Move this class to a separate file

Comment (matiasilveiro-gl) [reply lib/main.dart]: In fact, try to move each widget class to a separate file each!

File: lib/main.dart [hunk 31]
```diff
@@ -1,125 +1,218 @@
...
-          // wireframe for each widget.
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: <Widget>[
-            const Text(
-              'You have pushed the button this many times:',
+        body: Container(
+      color: Colors.black87,
+      child: Column(
+        children: <Widget>[
+          Image.asset(
```
Comment (matiasilveiro-gl) [inline lib/main.dart]: Format code

File: pubspec.yaml [hunk 32]
```diff
@@ -28,9 +28,12 @@ environment:
 # the latest version available on pub.dev. To see which dependencies have newer
 # versions available, run `flutter pub outdated`.
 dependencies:
+  shared_preferences: ^2.2.1
+  sqflite: ^2.3.0
+  provider: ^6.0.5
```
Comment (Kevin-Guillen) [inline pubspec.yaml]: do you need those dependencies?

File: lib/main.dart (hunk 8 above)
Comment (matiasilveiro-gl) [reply lib/main.dart]: Also, trailing comma hehe

File: lib/main.dart [hunk 33]
```diff
@@ -1,125 +1,218 @@
...
+                    side: const BorderSide(
+                        width: 5, color: Colors.indigo), // add a border
+                  ),
+                  color: Colors.blueAccent,
+                  child: Padding(
+                      padding: const EdgeInsets.all(15),
+                      child: Text(
+                        '${movie.voteAverage}',
+                        textScaleFactor: 2,
+                      )),
```
Comment (matiasilveiro-gl) [inline lib/main.dart]: Trailing comma

File: lib/main.dart [hunk 34]
```diff
@@ -1,125 +1,218 @@
...
+                  style: const TextStyle(
+                      color: Colors.white,
+                      fontWeight: FontWeight.bold,
+                      fontSize: 20.0),
+                ),
+              ),
+            ],
+          ),
+          SingleChildScrollView(
+            // this permit can be scroll
```
Comment (matiasilveiro-gl) [inline lib/main.dart]: Remove comments

File: lib/main.dart [hunk 35]
```diff
@@ -1,125 +1,218 @@
...
+  int like;
+
+  LikeButton(this.like, {super.key});
+
+  @override
+  State<LikeButton> createState() => _LikeButtonState();
+}
+
+class _LikeButtonState extends State<LikeButton> {
+  bool wasPress = false;
```
Comment (matiasilveiro-gl) [inline lib/main.dart]: Typo: `wasPressed`

File: lib/main.dart [hunk 36]
```diff
@@ -1,125 +1,218 @@
...
-        onPressed: _incrementCounter,
-        tooltip: 'Increment',
-        child: const Icon(Icons.add),
-      ), // This trailing comma makes auto-formatting nicer for build methods.
+    ));
+  }
+}
+
+class LikeButton extends StatefulWidget {
+  int like;
```
Comment (matiasilveiro-gl) [inline lib/main.dart]: I'd move this to the `State`, as that class is the one responsible of holding that value

File: pubspec.yaml (hunk 32 above)
Comment (matiasilveiro-gl) [inline pubspec.yaml]: Remove these imports

File: lib/main.dart [hunk 37]
```diff
@@ -18,201 +20,106 @@ class MyApp extends StatelessWidget {
...
                 ),
-              ),
-              Expanded(
-                child: Text(
-                  // Title
-                  movie.title,
-                  style: const TextStyle(
+                Expanded(
+                  child: Text(
+                    // Title
```
Comment (Kevin-Guillen) [inline lib/main.dart]: remove comment

File: lib/widget/like_button.dart [hunk 38]
```diff
@@ -0,0 +1,37 @@
...
+    setState(() {});
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return ElevatedButton.icon(
+      onPressed: () {
+        toggleLike();
+      },
+      icon: const Icon(Icons.thumb_up_outlined, color: Colors.black),
```
Comment (Kevin-Guillen) [inline lib/widget/like_button.dart]: comma

File: lib/widget/movie.dart [hunk 39]
```diff
@@ -0,0 +1,21 @@
...
+    required this.title,
+    required this.originalTitle,
+    required this.thumbnail,
+    required this.poster,
+    required this.overview,
+    required this.voteAverage,
+    required this.genres,
+    required this.countLikes,
+  });
+}
```
Comment (Kevin-Guillen) [inline lib/widget/movie.dart]: eof , and also do we need this class inside widget folder?

File: lib/widget/list_string_horizontal.dart [hunk 40]
```diff
@@ -0,0 +1,67 @@
+import 'package:flutter/material.dart';
+import 'package:the_movie_db/utils/general_constants.dart';
+
+class ListStringHorizontal extends StatelessWidget {
```
Comment (Kevin-Guillen) [inline lib/widget/list_string_horizontal.dart]: lets rename this class to something like GenresRow

File: lib/main.dart [hunk 41]
```diff
@@ -1,125 +1,125 @@
...
-
-  final String title;
-
-  @override
-  State<MyHomePage> createState() => _MyHomePageState();
-}
-
-class _MyHomePageState extends State<MyHomePage> {
-  int _counter = 0;
+class MovieDeck extends StatelessWidget {
```
Comment (matiasilveiro-gl) [inline lib/main.dart]: Move this class to a separate file

File: lib/main.dart [hunk 42]
```diff
@@ -1,125 +1,125 @@
...
-    setState(() {
-      // This call to setState tells the Flutter framework that something has
-      // changed in this State, which causes it to rerun the build method below
-      // so that the display can reflect the updated values. If we changed
-      // _counter without calling setState(), then the build method would not be
-      // called again, and so nothing would appear to happen.
-      _counter++;
-    });
-  }
+  const MovieDeck(this.movie, {super.key});
```
Comment (matiasilveiro-gl) [inline lib/main.dart]: Trailing comma

File: lib/main.dart [hunk 43]
```diff
@@ -1,125 +1,125 @@
...
+    poster: 'images/poster_mario_bros_movie.jpg',
+    overview: 'While working underground to fix a water main, Brooklyn '
+        'plumbers—and brothers—Mario and Luigi are transported down a '
+        'mysterious pipe and wander into a magical new world. But when the '
+        'brothers are separated, Mario embarks on an epic quest to find '
+        'Luigi.',
+    voteAverage: 7.8,
+    genres: ['Animation', 'Family', 'Adventure', 'Fantasy', 'Comedy'],
+    countLikes: 5,
+  );
```
Comment (FranciscoGaraventa) [inline lib/main.dart]: You can created a factory method that returns you this mock movie, so instead to created it on the ui side, you can have something like this:
`var movie = Movie.mockMovie();`
On movie class you are going to have something like:
`factory Movie.fromMock() => Movie(.....)`

File: lib/widget/like_button.dart [hunk 44]
```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/material.dart';
+
+class LikeButton extends StatefulWidget {
+  int like;
+
+  LikeButton(this.like, {super.key});
```
Comment (FranciscoGaraventa) [inline lib/widget/like_button.dart]: trailing comma

Comment (FranciscoGaraventa) [reply lib/widget/like_button.dart]: Also you can add all the parameter inside the {}

File: lib/main.dart (hunk 1 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/main.dart (hunk 2 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/main.dart (hunk 3 above)
Comment (SavoGK) [reply lib/main.dart]: done, i changed it to MovieDeck

File: lib/main.dart (hunk 4 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/main.dart (hunk 5 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/main.dart (hunk 6 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/main.dart (hunk 7 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/main.dart (hunk 8 above)
Comment (SavoGK) [reply lib/main.dart]: i created a file call "general_constants".  i resolved the constant value and the trailing comma

File: lib/main.dart (hunk 9 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/main.dart (hunk 10 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/main.dart (hunk 11 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/main.dart (hunk 12 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/main.dart (hunk 14 above)
Comment (SavoGK) [reply lib/main.dart]: For now, i only used this constants in main. For this reason, i created the constant in the same file but if in the future, i use in other thing i will put it in "general_constants"

File: lib/main.dart (hunk 15 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/main.dart (hunk 16 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/main.dart (hunk 17 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/main.dart (hunk 18 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/main.dart (hunk 19 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/main.dart (hunk 18 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/main.dart (hunk 20 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/main.dart (hunk 21 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/main.dart (hunk 22 above)
Comment (SavoGK) [reply lib/main.dart]: done, i removed it.

File: lib/main.dart (hunk 23 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/main.dart (hunk 24 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/main.dart (hunk 25 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/main.dart (hunk 26 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/main.dart (hunk 27 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/main.dart (hunk 35 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/main.dart (hunk 34 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/main.dart (hunk 33 above)
Comment (SavoGK) [reply lib/main.dart]: Done

File: lib/main.dart (hunk 37 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/widget/like_button.dart (hunk 38 above)
Comment (SavoGK) [reply lib/widget/like_button.dart]: done

File: lib/widget/movie.dart (hunk 39 above)
Comment (SavoGK) [reply lib/widget/movie.dart]: done

File: lib/widget/list_string_horizontal.dart (hunk 40 above)
Comment (SavoGK) [reply lib/widget/list_string_horizontal.dart]: done

File: lib/main.dart (hunk 41 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/main.dart (hunk 42 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/main.dart (hunk 43 above)
Comment (SavoGK) [reply lib/main.dart]: done

File: lib/widget/like_button.dart (hunk 44 above)
Comment (SavoGK) [reply lib/widget/like_button.dart]: done

File: lib/class/movie.dart
Comment (matiasilveiro-gl) [inline lib/class/movie.dart:1]: I'd rename the folder as `models`, `domain` or any other name than generic `class` name

File: lib/main.dart (hunk 41 above)
Comment (matiasilveiro-gl) [reply lib/main.dart]: Maybe you forgot to push?

Comment (SavoGK) [reply lib/main.dart]: sorry, i confused movie deck with movie. Now it´s done

**Review by Kevin-Guillen — APPROVED** [review]
lgtm

**Review by matiasilveiro-gl — APPROVED** [review]
LGTM! 🚀

**Review by FranciscoGaraventa — APPROVED** [review]

**Pull request merged**